tool/presubmit.dart

```diff
@@ -4,36 +4,39 @@ import 'dart:io';
 import 'package:path/path.dart' as p;
 
 main() {
+  // Change the CWD.
+  print('Changing CWD...');
+  Directory.current = p.join(p.current, 'workspace');
+
+  print('Running pub get...');
+  Process.runSync('pub', ['get']);
```

Comment:
This doesn't throw on fail. Maybe we need to create a helper to wrap this nicely – throw a good exception, etc.